src/invoke.rs

```diff
@@ -12,6 +21,8 @@ use crate::strval::{self, StrValError};
 pub struct Invoke {
     #[clap(long, parse(from_os_str))]
     file: std::path::PathBuf,
+    #[clap(long, parse(from_os_str))]
+    db_file: std::path::PathBuf,
```

Comment:
Let's use a single term to refer to the data being stored, either "storage" or "snapshot". If we use the term "db" here, and "snap" in another spot, and "storage" elsewhere, we are introducing three concepts to the reader when in fact there is only one concept.
```suggestion
    snapshot_file: std::path::PathBuf,
```